src/translationPipes/GenerateSound.ts

```diff
@@ -137,16 +137,22 @@ export class GenerateSound implements TranslationPipe {
                 audioConfig
             )
 
+            const childElement = /^<voice/.test(text)
```

Comment:
I think next time [startsWith function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) would be better here (few nano seconds faster? :D ).

Reply:
Good idea @nataliadiak please, can you make this small change? (New PR)

Reply:
Sure.

Reply:
I think we can leave it for future changes, it works same at this moment :)

But. We should probably do a text.trim() first if there is any accidental whitespace